dicomImport/matRad_importDicomCt.m

```diff
@@ -197,11 +198,18 @@
 ct.dicomInfo.Height                  = ctInfo(1).Height;
 ct.dicomInfo.RescaleSlope            = ctInfo(1).RescaleSlope;
 ct.dicomInfo.RescaleIntercept        = ctInfo(1).RescaleIntercept;
-if isfield(completeDicom,'PatientName')
+ct.dicomInfo.Manufacturer            = completeDicom.Manufacturer;
+ct.dicomInfo.ManufacturerModelName   = completeDicom.ManufacturerModelName;
+ct.dicomInfo.ConvolutionKernel       = completeDicom.ConvolutionKernel;
+
```

Comment:
I am somewhat confused: Is it possible that the fields Manufacturer, ManufacturerModelName and ConvolutionKernel dont exist? If yes shouldnt you also have a try catch environment here aswell - just like in calcWaterEqD?